src/main/kotlin/assertk/assertions/array.kt

```diff
@@ -31,25 +31,33 @@ fun <T> Assert<Array<T>>.hasSize(size: Int) {
 @JvmName("arrayContains")
 fun <T> Assert<Array<T>>.contains(element: Any?) {
     if (element in actual) return
-    expected("to contain ${show(element)} but was:${show(actual)}")
+    expected("to contain:${show(element)} but was:${show(actual)}")
```

Comment:
We might want to think about moving these messages into a method or something. We're repeating this a lot here.

Reply:
Ideas? I wanted to make messages pretty flexible so it's easy to change them to give more helpful info.

Reply:
As discussed, it's probably best to keep these pretty flexible. Approved 👍